src/App.js

```diff
@@ -12,6 +14,7 @@ function App() {
         <Switch>
           <Route path="/" exact component={MainContainer} />
           <Route path="/lessons" exact component={LessonsContainer} />
+          <Route path="/admin-app/students" exact component={StudentsPage} />
```

Comment:
Нужно убрать пробелы
```suggestion
          <Route path="/" exact component={MainContainer}/>
          <Route path="/lessons" exact component={LessonsContainer}/>
          <Route path="/admin-app/students" exact component={StudentsPage}/>
```